core-web/libs/block-editor/src/lib/extensions/ai-content-actions/ai-content-actions.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Observable, of } from 'rxjs';
+
+import { Component, EventEmitter, Output, OnInit } from '@angular/core';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+interface ActionOption {
+    label: string;
+    icon: string;
+    selectedOption: boolean;
+    callback: () => void;
+}
+
+@Component({
+    selector: 'dot-ai-content-actions',
+    templateUrl: './ai-content-actions.component.html',
+    styleUrls: ['./ai-content-actions.component.scss']
+})
+export class AIContentActionsComponent implements OnInit {
+    @Output() acceptEmitter = new EventEmitter<boolean>();
+    @Output() regenerateEmitter = new EventEmitter<boolean>();
+    @Output() deleteEmitter = new EventEmitter<boolean>();
```

Comment:
Maybe we can unify these three `@Output` into one called `@Output() actionEmitter` - Naming can be better - and emit the ActionType. Something like:

```ts
enum ACTIONS {
   ACCEPT = "ACCEPT",
   DELETE = "DELETE",
   REGENERATE = "REGENERATE"
}

@Output() actionEmitter = new EventEmitter<ACTIONS>();
```

That way the `@Ouput` will match the `Component Name` keeping consistency.